lib/Controller/WorkspaceController.php

```diff
@@ -124,7 +124,7 @@ public function folder(string $path = '/'): DataResponse {
 						'id' => $file->getId(),
 						'mimetype' => $file->getMimetype(),
 						'name' => $file->getName(),
-						'path' => $file->getPath()
+						'path' => $path . '/' . $file->getName()
```

Comment:
Just to be on the save side and not end up with double slashes:

```suggestion
						'path' => '/' . trim($path, '/') . '/' . $file->getName()
```

Reply:
Ah actually let's better use something like the following, since that then takes care about normalization of the path:

`$this->rootFolder->getUserFolder($this->userId)->getRelativePath($ile->getPath())`

Reply:
Yes... that is what i was looking for :heart_eyes: